library/src/main/java/com/novoda/downloadmanager/lib/DownloadStatus.java

```diff
@@ -13,6 +13,10 @@
 final class DownloadStatus {
 
     /**
+     * This download has been marked for deletion and it will be deleted in the future
+     */
+    public static final int DELETING = 188;
```

Comment:
I'm not sure about this one - but the DownloadManager has error codes as >400, does this count as an error or not? Discuss!